test/test_nn.py

```diff
@@ -16404,6 +16404,81 @@ def _helper(zero_infinity):
         _helper(zero_infinity=True)
         _helper(zero_infinity=False)
 
+    def _CTCLoss_no_batch_dim_helper(self, device):
```

Comment:
I think longer term the better approach would be to write a ModuleInfo entry for CTCLoss covering these cases, but I'm okay with these tests for now.